nextflow.config

```diff
@@ -17,13 +17,18 @@ params {
 
     // Pipeline options
     heatmap_genes_to_filter    = null
+    heatmap_id_column          = "SAMPLE_ID"
 
+    //Normalization options
+    normalization_method       = "GEO"
 
     // References
     genome                     = null
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = false
 
+    schema_ignore_params       = 'genome,genomes,igenomes_base'
```

Comment:
you don't need this params anymore since it's replaced by `validationSchemaIgnoreParams`